src/bin/diff-folded.rs

```diff
@@ -0,0 +1,64 @@
+use env_logger::Env;
+use std::io;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+use inferno::diff_folded::{self, Options};
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "inferno-diff-folded", author = "")]
+struct Opt {
```

Comment:
I'd like to see some of the help docs from [upstream](https://github.com/brendangregg/FlameGraph/blob/1b1c6deede9c33c5134c920bdb7a44cc5528e9a7/difffolded.pl#L10L26) printed both by the CLI and in the Rust library docs.

Reply:
There are also a few lines [here](https://github.com/brendangregg/FlameGraph/blob/1b1c6deede9c33c5134c920bdb7a44cc5528e9a7/difffolded.pl#L68L70) that may be good to include